### ISSUE TITLE
Train Open-world object detection on custom dataset

### ISSUE THREAD

**Issue by kojingying** [mmaaz60/mvits_for_class_agnostic_od#26]
Hi, thanks for the great work!

I have a few questions regarding to open world object detection:
1. how to fine tune the the pertained model for open world object detection?
2.  how should I prepare the data?
3.  do you provide test/inference and evaluation script for open world object detection?

**Comment by mmaaz60** [comment]
Hi @kojingying,

Thank you for your interest in our work. Please note the following,

We use MAVL (MDef-DETR) class-agnostic proposals to generate pseudo labels for unknowns in [ORE](https://openaccess.thecvf.com/content/CVPR2021/papers/Joseph_Towards_Open_World_Object_Detection_CVPR_2021_paper.pdf). We did not finetune our model directly for Open-world task, instead we propose to improve ORE by generating unknown pseudo labels using our model.

We provide details of Open-world object detection application using ORE in Sections 4.2 and A.4 of our [paper](https://arxiv.org/pdf/2111.11430.pdf). The detailed instructions to generate unknown pseudo labels and the ORE pretrained models are available at [applications/README.md](https://github.com/mmaaz60/mvits_for_class_agnostic_od/tree/main/applications).

I hope this will help. Please feel free to ask if you have any questions. Thanks
